sdk/streamlib-python-wheel/tests/test_encoded_video_frame_cast.py

```
@@ -0,0 +1,281 @@
+ANNEX_B_ACCESS_UNIT = b"\x00\x00\x00\x01\x65\x88"
+
+BT709_COLOR_ON_THE_WIRE = {
+    "primaries": "bt709",
+    "transfer": "bt709",
+    "matrix": "bt709",
+    "range": "limited",
+}
+
+
+def encoded_frame_bag(**overrides: Any) -> "dict[str, Any]":
+    """A sync-point H.264 access unit at the coded extent of a 320×180 source."""
```

Ruff (0.16.3)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)